index.js

```diff
@@ -177,10 +178,15 @@ Parser.parseString = function(xml, callback) {
   });
 }
 
-Parser.parseURL = function(url, callback) {
+Parser.parseURL = function(feedUrl, callback) {
   var xml = '';
-  var get = url.indexOf('https') === 0 ? HTTPS.get : HTTP.get;
-  var req = get(url, function(res) {
+  var get = feedUrl.indexOf('https') === 0 ? HTTPS.get : HTTP.get;
+  var parsedUrl = url.parse(feedUrl);
+  var req = get({
+    hostname: parsedUrl.hostname,
+    path: parsedUrl.path,
```

Comment:
The 'path' attribute will contain the 'search' string, if present. https://nodejs.org/api/url.html#url_urlobject_path

> On Jul 11, 2016, at 10:20, Bobby Brennan <notifications@github.com> wrote:
> 
> In index.js:
> 
> >    var xml = '';
> > -  var get = url.indexOf('https') === 0 ? HTTPS.get : HTTP.get;
> > -  var req = get(url, function(res) {
> > +  var get = feedUrl.indexOf('https') === 0 ? HTTPS.get : HTTP.get;
> > +  var parsedUrl = url.parse(feedUrl);
> > +  var req = get({
> > +    hostname: parsedUrl.hostname,
> > +    path: parsedUrl.path,
> Can you add parsedUrl.search to the path if it exists? Otherwise LGTM
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>